addon/components/validated-input/types/checkbox-group.js

```diff
@@ -0,0 +1,23 @@
+import Component from "@ember/component";
+import { action } from "@ember/object";
+
+import layout from "../../../templates/components/validated-input/types/checkbox-group";
+
+export default Component.extend({
+  layout,
+
+  @action
+  onUpdate(key) {
+    const value = this.value || [];
+    const index = value.indexOf(key);
+
+    if (index > -1) {
+      value.splice(index, 1);
+    } else {
+      value.push(key);
+    }
+
+    this.update(value);
```

Comment:
Where does `this.update` come from? I couldn't find anything in the component docs am i missing something obvious here?

Reply:
The public API is called `on-update` - it is defined here: https://github.com/adfinis-sygroup/ember-validated-form/blob/b8688b6d9dedbd9096d34c3b236bf59efb045556/addon/components/validated-input.js#L90